test/src/pumps_test.dart

```diff
@@ -37,6 +37,16 @@ extension on CommonFinders {
 }
 
 void main() {
+  final imageCache = ImageCache();
+
+  // We need to clear out images from the cache,
+  // which requires us to initialize a custom [WidgetsBinding].
+  AlchemistWidgetsBinding(imageCache: imageCache);
```

Comment:
and this is where we could call `ensureInitialized(imageCache)` to point out to everyone we're using a custom binding override, since constructors really shouldn't be relied on to cause side effects (even when they do).

Reply:
resolving these as @Kirpal helped me refactor to use the imageCache from PaintingsBinding 😄